Assets/Scripts/Auction/Auctioneer.cs

```diff
@@ -65,6 +67,7 @@ public void Sell()
         audioSource.Stop();
         audioSource.clip = sold.RandomElement();
         audioSource.Play();
+        audioSourceFX.PlayDelayed(0.5f);
```

Comment:
This delay seems just a little too long, perhaps shorten it by 100 ms?

Reply:
Sure :)